Misc/Positions_Area Positions 4 External.txt

```diff
@@ -33,10 +33,14 @@ LFEE_N_CTR:Reims Control:127.550:EEN:C:LFEE:CTR:LFEE:NORTH:5700:5767
 LFEE_CTR:Reims Control:128.300:EE:C:LFEE:CTR:LFEE:CTR:5700:5767
 
 LFRR_CTR:Brest Control:125.500:BR:C:LFRR:CTR:LFRR:CTR:5640:5657
+LFRR_A_CTR:Brest Control:131.270:BRA:C:LFRR:CTR:LFRR:CTR:5640:5657
```

Comment:
```suggestion
```

Reply:
Is this because the sector doesn't border the UK so is not required?

Reply:
Yup, that's my thinking. Generally we don't define unless it's relevant